### DISCUSSION TITLE
How can I help?

### DISCUSSION THREAD

**Discussion by cosminj** [johnkeel-thork/TET#8] [General]
Hello,
As a developer, I'd like to help out but I'm not sure what to begin with.
I can do backends and some iOS/Android/DB as well.
Are you going to use Issues? Or perhaps a backlog in Discussions?

**Comment by johnkeel-thork** [comment]
I just setup a small text about the App objectives:
https://github.com/johnkeel-thork/TET/discussions/11

**Comment by yunusde** [comment]
I am here too, mostly on Server Side (Backend and Web) and a bit on Android. So lets get coding :)

**Comment by Ola-Palm** [comment]
Hello. I'm a developer knowledge in realtime communications weight at backend and I would like to help, if I can. 
iOS, swift, golang, docker, Jenkins, typeScript. 

I really think that a iOS version would be nice. Perhaps a dart/flutter application that would run on both platforms?
